WordPress/Classes/ViewRelated/Notifications/Tools/NotificationMediaDownloader.swift

```diff
@@ -235,7 +231,11 @@ import AFNetworking
 
     // MARK: - Private Properties
     private let responseSerializer  = AFImageResponseSerializer()
-    private let downloadQueue       = NSOperationQueue()
+    private lazy var downloadSession:AFURLSessionManager = {
```

Comment:
I'm not sure what the standard is, but i _think_ we're having no spaces before the `:` and a single one after?

```
downloadSession: AFURLSessionManager
```

Please, correct me if that's wrong!